tests/backup/backup_test.go

```diff
@@ -6653,15 +6655,17 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			schPolicyUid, _ = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicPolicyName)
 			backupName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, timeStamp)
-			schBackupName, err = CreateScheduleBackup(backupName, SourceClusterName, backupLocationName, backupLocationUID, bkpNamespaces,
+			err = CreateSchedule(backupName, SourceClusterName, backupLocationName, backupLocationUID, bkpNamespaces,
 				labelSelectors, orgID, "", "", "", "", periodicPolicyName, schPolicyUid, ctx)
```

Comment:
same

Reply:
I overlooked the need to rename the "backupName" variable to "scheduleName" before passing it to the "CreateSchedule" function.  I will rectify it.